### PR TITLE
Add support for `SetupFutureUsage` on `PaymentIntent` update and confirm

### PR DESCRIPTION
**Pull request by remi-stripe**
Waiting for this to be ready in the API

cc @garrett-stripe @stripe/api-libraries-releasers

### CODE REVIEW

**Comment by tobecon** [comment]
Could you also add this property to the  [SessionPaymentIntentDataOptions](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Checkout/SessionPaymentIntentDataOptions.cs) ?

File: src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs [hunk 1]
```diff
@@ -60,9 +60,27 @@ public class PaymentIntentConfirmOptions : BaseOptions
...
+        /// PaymentIntent. You will then be able to make later off-session payments for this
+        /// customer.
+        /// </summary>
+        [JsonProperty("setup_future_usage")]
+        public string SetupFutureUsage { get; set; }
+
         /// <summary>
         /// Shipping information for this PaymentIntent.
         /// </summary>
+
```
Comment (ob-stripe) [inline src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs]: Unnecessary newline here.

File: src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs [hunk 2]
```diff
@@ -60,9 +60,27 @@ public class PaymentIntentConfirmOptions : BaseOptions
         [JsonProperty("save_payment_method")]
         public bool? SavePaymentMethod { get; set; }
 
+        /// <summary>
+        /// Indicates that you intend to make future payments with this PaymentIntent’s payment
+        /// method.
```
Comment (ob-stripe) [inline src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs]: Feel free to ignore, but if you want this comment to be rendered as different paragraphs, you must wrap each paragraph in a `<para></para>` tag.

Comment (remi-stripe) [reply src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs]: Yeah I think I'll ignore for this one but noted.

**Comment by remi-stripe** [comment]
@ob-stripe PTAL

**Pull request merged**